i18n/src/closureI18nExtractor.js

```diff
@@ -58,10 +60,12 @@ function extractCurrencySymbols(content) {
 
 function extractDateTimeSymbols(content, localeInfo) {
   //eval script in the current context so that we get access to all the symbols
+  // eslint-disable-next-line no-eval
   eval(content.toString());
   for (var propName in goog.i18n) {
     var localeID = findLocaleId(propName, 'datetime');
     if (localeID) {
+      // eslint-disable-next-line no-unused-vars
```

Comment:
Instead of this, you could change the `localeInfo[localeID].DATETIME_FORMATS...` to `info.DATETIME_FORMATS...`.